### PR TITLE
DEV-10000 award profile changes for cd

### PR DESCRIPTION
**Pull request by nick-torres**
**High level description:**
award profile changes - unhide tooltips, move cd to new row in accordion 

**JIRA Ticket:**
[DEV-10000](https://federal-spending-transparency.atlassian.net/browse/DEV-10000)

**Mockup:**
https://app.zeplin.io/project/61f8076f4ef77f11fcc37f55/screen/63e3ce16df2834197a6c54cf

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [ ] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/js/components/award/shared/additionalInfo/AdditionalInfo.jsx [hunk 1]
```diff
@@ -141,6 +153,18 @@ export default class AdditionalInfo extends React.Component {
...
         const { overview } = this.props;
         const data = this.data();
+
+        const cdPOP = data.placeOfPerformance.Address.data.pop().trim();
+        const splitCDPOP = cdPOP.split(": ");
+        if (splitCDPOP.length === 2) {
+            data.placeOfPerformance["Congressional District"] = splitCDPOP[1];
+        }
+
+        const cdRD = data.recipientDetails["Congressional District"].data.pop().trim();
```
Comment (brianpetway) [inline src/js/components/award/shared/additionalInfo/AdditionalInfo.jsx:163]: When this block is left in, it looks like the CD listed here is actually the Place of Performance CD; if it's commented out, the CD listed here changes to match the Recipient Address, but it's formatting is different than the CD in the Place of Performance section; I was using 
CONT_AWD_DEAC0494AL85000_8900_-NONE-_-NONE-
to test.

**Comment by brianpetway** [comment]
Two inline comments about the CD on Award Profile pages.

But also, on the State page, in the map filter, the placement of the CD tooltip is a little weird. It goes off the right edge at screens from 1400-1600, then between 1000 and 1400 sometimes the little pointer on the tooltip is on the left and sometimes it's in the top, when the tooltip is below the info icon

File: src/js/dataMapping/award/additionalDetailsContract.js [hunk 2]
```diff
@@ -137,10 +137,13 @@ const additionalDetailsContracts = (awardData) => {
                 data: [
                     `${recipient.location.streetAddress}`,
                     `${recipient.location.regionalAddress}`,
-                    `${recipient.location._country}`,
-                    `${recipient.location.fullCongressionalDistrict}`
+                    `${recipient.location._country}`
                 ]
             },
+            'Congressional District': {
```
[the remark this answers is not in the document]
Comment (nick-torres) [reply src/js/dataMapping/award/additionalDetailsContract.js:146]: it is not

**Comment by nick-torres** [comment]
your last comment should be part of 9949

**Review by brianpetway — APPROVED** [review]

**Pull request merged**